src/components/Personality.js

```diff
@@ -30,10 +33,52 @@ class Personality extends Component {
         ['Traditional', 'Free Thinking'],
       ],
     };
+
+    ServerUtil.initCallback(
+      (result) => this.onRequestSuccess(result),
+      (error) => this.onRequestFail(error)
+    );
   }
 
-  render() {
+  onRequestSuccess(result) {
+    console.log(result);
```

Comment:
Is this for debugging? 
looks like it will be replace with real data later.
If so, leave a comment